apps/os/backend/org-utils.ts

```diff
@@ -9,6 +11,24 @@ import { getUserOrganizations } from "./trpc/trpc.ts";
 import { getOrCreateAgentStubByName } from "./agent/agents/stub-getters.ts";
 import { createStripeCustomerAndSubscriptionForOrganization } from "./integrations/stripe/stripe.ts";
 
+export const createGithubRepoInEstatePool = async (orgName?: string) => {
```

Comment:
The org name shouldn't be optional. In fact, we should probably be passing in more info about the estate, so that we can link to it in the repo some day. Fine for now to just use the org name though I guess